web3c/oasis.js

```diff
@@ -221,12 +233,17 @@ class OasisUtils {
   }
 }
 
-function getPublicKeyOutputFormatter (t) {
+function getPublicKeyOutputFormatter (t, options) {
   // We called oasis_getPublicKey on a non-confidential contract.
   if (!t) {
     return t;
   }
-  let signer = new Signer(KeyManager.publicKey());
+  let signer = undefined;
+  if (options && options.keyManagerPublicKey) {
+    signer = new Signer(options.keyManagerPublicKey);
```

Comment:
lets put this in readme documentation?

do we need to have this defined as a constant string `const keyManagerPublicKeyOption = 'keyManagerPublicKey';` so that it doesn't get mangled by minimization scripts?

Reply:
Indeed lets put it in documentation.

I don't know if we need the constant string. This would make me sad. Any docs to point to on the mechanism you're referring to? If not, I say we just don't include it. If it breaks, we'll know and we'll know why.

Reply:
e.g. https://github.com/mishoo/UglifyJS2 has the option `--mangle-props keep_quoted` to allow the processor to minify property names when used as a `.prop` but not when as `['prop']`. This is a somewhat common distinction, but since we aren't using a minified it doesn't really matter at the moment